src/ETH/Accounts.md

```diff
@@ -0,0 +1,59 @@
+# P15 - Accounts
+
+## Account-Based Ledger
+Comparing Bitcoin and Ethereum:
+- Bitcoin uses a **transaction-based ledger**, where account balances are inferred through `UTXO`.
+- Ethereum uses an **account-based ledger**, similar to a banking system, where accounts directly store `balance` states.
+
+Differences in transfer mechanisms:
+- Bitcoin: When transferring from account B to C, the source of coins must be specified, and every `UTXO` must be entirely spent at once, so any remaining coins should be transferred back to another self-owned account.
+- Ethereum: The source of coins during a transfer is the `balance` state of the account, as the account includes the `balance` state, so it does not need to be entirely spent at once.
+
+```txt
+         Transaction-based Ledger                    Account-Based Ledger             
+ ┌────────────────────────────────────────┐     ┌───────────────────────────────────┐ 
+ │      ┌──────────────────┐              │     │                                   │ 
+ │      │                  │              │     │                                   │ 
+ │┌─────▼───────┐         ┌┼─────────────┐│     │┌─────────────┐    ┌─────────────┐ │ 
+ ││A───►B(10BTC)│         │B─┬──►C(3BTC) ││     ││A───►B(10ETH)│    │B───►C(3ETH) │ │ 
+ ││             │         │  │           ││     ││             │    │             │ │ 
+ ││             │         │  └──►B'(7BTC)││     ││             │    │             │ │ 
+ │└─────────────┘         └──────────────┘│     │└─────────────┘    └─────────────┘ │ 
+ └────────────────────────────────────────┘     └───────────────────────────────────┘               
```

Comment:
you sure?

Reply:
for sure. The arrow from B to A indicates the source of the coin, which is feature of the UTXO model but unnecessary for the account-based model.